README.rst

```diff
@@ -818,7 +730,76 @@ host machine.
 
 .. code:: powershell
 
-   sam local invoke --docker-volume-basedir /c/Users/shlee322/projects/test "Ratings"
+   $ sam local invoke --docker-volume-basedir /c/Users/shlee322/projects/test "Ratings"
+
+Advanced installations
+----------------------
+
+Build From Source
+~~~~~~~~~~~~~~~~~
+
+First, install Python(2.7) on your machine, then run the following:
```

Comment:
Shouldn't this now be 2.7 or 3.6?

https://github.com/awslabs/aws-sam-cli/blob/develop/README.rst#build-from-source

Reply:
done